src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java

```diff
@@ -1,30 +1,102 @@
+                if (messageText.equals("/aries")) {
+
+                    return sendMessage.setText(aries.forAries("Овен", 0) + aries.forAries("Овен",1)
+                            + aries.forAries("Овен",2) + aries.forAries("Овен",3));
+                }
+
+
+                if (messageText.equals("/cancer")) {
+                    return sendMessage.setText(cancer.forCancer("Рак", 0) + cancer.forCancer("Рак", 1) +
+                            cancer.forCancer("Рак", 2) + cancer.forCancer("Рак", 3));
                 }
+                if (messageText.equals("/capricorn")) {
+                    return sendMessage.setText(capricorn.forCapricorn("Козерог", 0) + capricorn.forCapricorn("Козерог", 1) +
+                            capricorn.forCapricorn("Козерог", 2) + capricorn.forCapricorn("Козерог", 3));
+                }
+                if (messageText.equals("/gemini")) {
+                    return sendMessage.setText(gemini.forGemini("Близнецы", 0) + gemini.forGemini("Близнецы", 1)
+                            + gemini.forGemini("Близнецы", 2) + gemini.forGemini("Близнецы", 3));
+                }
+
+                if (messageText.equals("/libra")) {
+                    return sendMessage.setText(libra.forLibra("Весы", 0) + libra.forLibra("Весы", 1) +
+                            libra.forLibra("Весы", 2) + libra.forLibra("Весы", 3));
+                }
+                if (messageText.equals("/pisces")) {
+                    return sendMessage.setText(pisces.forPisces("Рыбы", 0) + pisces.forPisces("Рыбы", 1)
+                            + pisces.forPisces("Рыбы", 2) + pisces.forPisces("Рыбы", 3));
+                }
+                if (messageText.equals("/saggitarius")) {
+                    return sendMessage.setText(saggittarius.forSaggitarius("Стрелец", 0) + saggittarius.forSaggitarius("Стрелец", 1)
+                            + saggittarius.forSaggitarius("Стрелец", 2) + saggittarius.forSaggitarius("Стрелец", 3));
+                }
+
+                if (messageText.equals("/scorpio")) {
+                    return sendMessage.setText(scorpio.forScorpio("Скорпион", 0) + scorpio.forScorpio("Скорпион", 1)
+                            + scorpio.forScorpio("Скорпион", 2) + scorpio.forScorpio("Скорпион", 3));
+                }
+                if (messageText.equals("/taurus")) {
+                    return sendMessage.setText(taurus.forTaurus("Телец", 0) + taurus.forTaurus("Телец", 1)
+                            + taurus.forTaurus("Телец", 2) + taurus.forTaurus("Телец", 3));
+                }
+                if (messageText.equals("/virgo")) {
+                    return sendMessage.setText(virgo.forVirgo("Дева", 0) + virgo.forVirgo("Дева", 1) +
+                            virgo.forVirgo("Дева", 2) + virgo.forVirgo("Дева", 3));
+                }
+
             }
         }
-        return sendMessage.setText("sorry, i don`t understand you");
+        return sendMessage.setText("I don't know this horoscope!");
```

Comment:
this message - the answer from bo to client for unknown comand to bot. When i send to the bot command "/setings" - i take this message : "I don't know this horoscope!". this not good

Reply:
I fixed it!